modules/administration-guide/pages/installing-che-on-minikube-keycloak-oidc.adoc

```diff
@@ -0,0 +1,313 @@
+:_content-type: PROCEDURE
+:navtitle: Installing {prod-short} on Minikube with {identity-provider} as OIDC Identity Provider
+:description: Installing {prod-short} on Minikube with {identity-provider} as OIDC Identity Provider
+:keywords: overview, installing-che-on-minikube-keycloak-oidc
+:page-aliases: installation-guide:installing-che-on-minikube-keycloak-oidc, overview:installing-che-on-minikube-keycloak-oidc
+
+
+[id="installing-{prod-id-short}-on-minikube-keycloak-oidc_{context}"]
+= Installing {prod-short} on Minikube with {identity-provider} as OIDC Identity Provider
+
+This article explains how to create a single-node {kubernetes} cluster with Minikube to deploy {prod-short} and
+configure it to use {identity-provider} as OIDC identity provider.
```

Comment:
Since this article is specific to minikube + keycloak, does it make sense to avoid the template fields? With the wrong values, this could turn into an article that refers to minikube with OpenShift version X or higher, using `oc` instead of `kubectl`.

Reply:
We have a similar doc about deploy Eclipse Che on minikube [1] with templates instead of hardcoded values.
So, I think we don't have to care much.
/cc @max-cx 

[1] https://github.com/eclipse-che/che-docs/blob/70783006917dab18651d0e9a274b98b0a176c47b/modules/administration-guide/pages/installing-che-on-minikube.adoc#L39-L38